src/service/url-expander/expander.js

```diff
@@ -193,13 +214,43 @@ export class Expander {
     return evaluateNextLevel();
   }
 
+
+  /**
+   * Called when a binding is ready to be resolved. Looks to
+   * @param {Object<string, *>} bindingInfo An object containing the name of macro
+   * and value to be resolved.
+   * @param {Array=} opt_args Arguments passed to the macro.
+   * @param {*=} opt_sync Whether the binding should be resolved synchronously.
+   */
+  evaluateBinding_(bindingInfo, opt_args, opt_sync) {
+    let binding;
+    if (bindingInfo.optional) {
```

Comment:
sorry my bad. optional is a bad name. bindingOverride might be a better name. opt_bindings simply means the arg is optional to the function.

Reply:
I want to call it something like `passed` or `given` but I can't come up with the right word.